assets/package.json

```diff
@@ -57,13 +57,14 @@
     "react": "^16.4.1",
     "react-datepicker": "^4.10.0",
     "react-dom": "^16.4.1",
+    "react-hook-form": "^7.43.5",
     "source-map-loader": "^4.0.1",
     "ts-jest": "^29.0.5",
     "ts-loader": "^9.4.2",
     "typescript": "^4.9.5",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.1",
-    "yup": "^1.0.2"
+    "yup": "^0.32"
```

Comment:
Why the downgrade?

Reply:
yup 1.0 was released fairly recently and had breaking changes that the `@hookform/resolvers` doesn't support yet